webapp/app/forms/fields.py

```diff
@@ -96,7 +96,10 @@ def __call__(self, field, **kwargs):
             if self.sub_field_separator and idx < len(self.input_field_lengths) - 1:
                 joined_input_fields += Markup(self.sub_field_separator)
 
-        return Markup(joined_input_fields)
+        return Markup(render_template('fields/multiple_input_field.html',
+                                      joined_input_fields=joined_input_fields,
+                                      name=field.id,
+                                      input_field_lengths=self.input_field_lengths))
```

Comment:
Why do you add `name` and `input_field_lengths` as parameters? I don't think you're using them in the template?

Reply:
Good catch, input_field_length is now set as an additional attribute of the subfield and can go. `name` is used in line 6 in the template: `const sub_fields = document.querySelectorAll('input[name={{ name }}]');`